File: C-Module/Classwork/poc/leapyear.c
// Year is leap or not

#include <stdio.h>

void main()
{
	int year = 2004;
	if(year % 4 ==0 && year%100 != 0 || year%100 == 0)
	{
		printf("Year is leap year");
	}
	else
	{
		printf("No leap year");
	}
}

File: Java-Module/Assignmets/Assignment_1/leapyear.java
class leapyear
{	
	public static void main(String args[])
	{
		int year = 2100;
		if(year%4==0 && year%100 != 0 || year%400 == 0)
			System.out.printf("This is a leap year");
		else 
			System.out.printf("Not a leap year");
	}
	
}